release/src/main/scripts/run_rc_validation.sh

```diff
@@ -0,0 +1,338 @@
+  read USER_SERVICE_ACCOUNT_NAME
+  SERVICE_ACCOUNT_KEY_JSON=${USER}_json_key.json
+  gcloud iam service-accounts keys create ${SERVICE_ACCOUNT_KEY_JSON} --iam-account ${USER_SERVICE_ACCOUNT_NAME}@${USER_GCP_PROJECT}
+  export GOOGLE_APPLICATION_CREDENTIALS=pwd/${SERVICE_ACCOUNT_KEY_JSON}
+
+  echo "-------------------Creating BigQuery Dataset-----------------------"
+  bq mk --project=${USER_GCP_PROJECT} ${MOBILE_GAME_DATASET}
+
+  echo "----------------------Creating Pubsub Topic------------------------"
+  gcloud alpha pubsub topics create --project=${USER_GCP_PROJECT} ${MOBILE_GAME_PUBSUB_TOPIC}
+
+  echo "[Current task] Java mobile game validations: UserScore, HourlyTeamScore, Leaderboard"
+  ./gradlew :beam-runners-google-cloud-dataflow-java:runMobileGamingJavaDataflow \
+  -Prepourl=${REPO_URL} \
+  -Pver= ${RELEASE} \
+  -PgcpProject=${USER_GCP_PROJECT} \
+  -PgcsBucket=gs://dataflow-samples/game/gaming_data1.csv \
+  -PbqDataset=${MOBILE_GAME_DATASET} -PpubsubTopic=${MOBILE_GAME_PUBSUB_TOPIC}
+
+  echo "-------------------Cleaning Up BigQuery Dataset-------------------"
+  bq rm -rf --project=${USER_GCP_PROJECT} ${MOBILE_GAME_DATASET}
+fi
+
+echo "==================Starting Python Quickstart and MobileGame==================="
+echo "This task will create a PR against apache/beam, trigger a jenkins job to run:"
+echo "1. Python quickstart validations(batch & streaming)"
+echo "2. Python MobileGame validations(UserScore, HourlyTeamScore)"
+echo "[Confirmation Required] Do you want to proceed? [y|N]"
+read confirmation
+if [[ $confirmation = "y" ]]; then
+  echo "[Input Required] Please enter your github repo URL forked from apache/beam:"
+  read USER_REMOTE_URL
+  WORKING_BRANCH=python_validatoin_pr
+  git checkout -b ${WORKING_BRANCH}
+  touch empty_file.txt
+  git add empty_file.txt
+  git commit -m "Add empty file in order to create PR"
+  git push -f ${USER_REMOTE_URL}
+  hub pull-request -b apache:master -F- <<<"[DO NOT MERGE]Run Python RC Validation Tests
+
+
+  Run Python ReleaseCandidate"
+
+  echo "[NOTE] If there is no jenkins job started, please comment generated PR with: Run Python ReleaseCandidate"
+fi
+
+echo "==============Starting Python Leaderboard & GameStates Validations==============="
+echo "This task asks for GCP resources. Do you want to proceed? [y|N]"
+read confirmation
+if [[ $confirmation = "y" ]]; then
+  cd ~/${LOCAL_CLONE_DIR}
+
+  echo "---------------------Downloading Python Staging RC----------------------------"
+  wget -r --no-paraent -A 'apache-beam-${RELEASE}.zip*' ${PYTHON_RC_DOWNLOAD_URL}/${RELEASE}/python/
+  ls | grep 'apache-beam-${RELEASE}'
+
+  echo "--------------------------Verifying Hashes------------------------------------"
+  sha512sum -c apache-beam-${RELEASE}.zip.sha512
+
+  echo "----------------------------Building Python SDK-------------------------------"
+  sudo apt-get install unzip
+  unzip apache-beam-${RELEASE}.zip
+  cd apache-beam-${RELEASE}
+  python setup.py sdist
```

Comment:
Why do we build as part of validation?

Reply:
Addressed in commit https://github.com/apache/beam/pull/6081/commits/a2303400f6fad8c9cda22395be24697bb4f1efed